.github/workflows/rw-publish-prerelease.yml

```diff
@@ -40,33 +40,47 @@ jobs:
         with:
           node-version: 18.17.0
 
-      - name: Get version
+      - name: Get version from branch ${{ inputs.source-branch }}
         uses: gooddata/gooddata-ui-sdk/.github/actions/get-version-action@master
         id: get-version
 
-      - name: Check version
+      - name: Check version on ${{ inputs.source-branch }} branch
         run: |
-          if [[ ! $VERSION =~ ^[0-9]+\.[0-9]+\.[0-9]+-(beta|alpha)\.[0-9]+$ ]]; then
-            echo "The version "$VERSION" does not match X.Y.Z-beta.W or X.Y.Z-alpha.W"
-            exit 1
+          if [[ "${BRANCH}" == "release" ]]; then
+            # we need check that prerelase will not run on unfinished release where packages published
+            # but release process not finished yet (branch is not deleted)
+            # when we prerelease from rel/X.Y branch, we are processing patch and its ok becouse bump will
+            # correctly bump version to next patch
+
+            if [[ ! $VERSION =~ ^[0-9]+\.[0-9]+\.[0-9]+-(beta|alpha)\.[0-9]+$ ]]; then
+              echo "The version "$VERSION" does not match X.Y.Z-beta.W or X.Y.Z-alpha.W"
+              exit 1
+            fi
           fi
         env:
           VERSION: ${{ steps.get-version.outputs.version }}
+          BRANCH: ${{ inputs.source-branch }}
 
   setup-params:
     needs: [check-version]
     runs-on: [ubuntu-latest]
     outputs:
       prerelease-id: ${{ steps.validate-params.outputs.prerelease-id }}
-
+      prerelease-tag: ${{ steps.validate-params.outputs.prerelease-tag }}
     steps:
       - name: validate params
         id: validate-params
         run: |
-          if [ "${{ inputs.source-branch }}" = "master" ]; then
+          # TODO: remove inputs from bash script, use env
```

Comment:
should be easy to fix, can you please move to env

Reply:
done